pkg/images/defaults/flavor.go

```diff
@@ -24,6 +25,19 @@ type imageFlavorDescriptor struct {
 }
 
 var (
+	imageFlavorMainRegistry = env.RegisterSetting(
+		"ROX_MAIN_REGISTRY",
+		env.WithDefault("quay.io/rhacs-eng"),
+	)
+	imageFlavorScannerImageTag = env.RegisterSetting(
+		"ROX_SCANNER_VERSION",
+		env.WithDefault(version.GetAllVersionsDevelopment().ScannerVersion),
+	)
+	imageFlavorMainImageTag = env.RegisterSetting(
+		"ROX_MAIN_VERSION",
+		env.WithDefault(version.GetAllVersionsDevelopment().MainVersion),
+	)
```

Comment:
I think at least two of these new vars may be redundant.

I can already set the equivalent of your `ROX_MAIN_VERSION` with the existing env called `MAIN_IMAGE_TAG`.
Here is an example I used recently,
```
export MAIN_IMAGE_TAG=3.73.x-591-g3cfa857718
```

I have personally never have to change the main registry or scanner version, 
but I think that your `ROX_MAIN_REGISTRY` already has an equivalent in `MAIN_IMAGE_REPO`,
and your `ROX_SCANNER_VERSION` already has an equivalent in `SCANNER_IMAGE`.

See this script for example usage:
https://github.com/stackrox/stackrox/blob/master/deploy/common/deploy.sh

Reply:
@vjwilson I've used `MAIN_IMAGE_TAG` variable when using `deploy.sh`, however for a use case that runs the operator locally (e.g. make install run) I don't think it is available.